modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -525,6 +525,19 @@ Note: you must use json to write multi array values.
 | `--output=[OUTPUT]` | The output format to use (`plain`, `json` or `json_pretty`, default is `plain`).
 |===
 
+Examples
```

Comment:
Why not format this as a header?

Reply:
I asked this exact question in chat. I personally did not do it because it's not done in other examples, so I did not want to go against your rules.

But, I find it better to format this as a header so than we can link to in in other parts of the docs.

Reply:
Fine by me. Thanks for the clarification.